src/tools/miri/src/shims/intrinsics/mod.rs

```diff
@@ -84,24 +100,6 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
         }
 
         match intrinsic_name {
-            // Miri overwriting CTFE intrinsics.
```

Comment:
This comment was never correct. Miri does not oeverwrite CTFE intrinsics, these intrinsics's CTFE impls are not in any code path that miri could possibly execute.